introduction.md

```diff
@@ -0,0 +1,7 @@
+* John's comments for the fiest excercise in git hub
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.